evergreen.yml

```diff
@@ -55,22 +55,13 @@ buildvariants:
   - amazon2-build
   expansions:
     distro: amazon2
-    # latest successful build from master as of 2020-10-01 on "Amazon Linux 2" compile task "Binaries" file
+    # latest successful build from master as of 2022-06-29 on "Amazon Linux 2" compile task "Binaries" file
     # commit should match that in get-mongo-source.sh
     mongodb_archive_url:
-      https://mciuploads.s3.amazonaws.com/mongodb-mongo-master/amazon/cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3/binaries/mongo-mongodb_mongo_master_amazon_cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3_21_12_03_09_47_48.tgz
+      https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-amazon2-6.0.0.tgz
   tasks:
   - name: tg_compile_and_test_with_server
 
-- name: archlinux
```

Comment:
Bump about asking about these variants in #performance-tooling-users! I think @mvankeulen94 was also looking into discussions about genny platform support, and might want to have a say? I might be misremembering though.

Reply:
archlinux is not under [supported platforms](https://www.mongodb.com/docs/manual/installation/#supported-platforms) for 6.0.